packages/svgs/package.json

```diff
@@ -0,0 +1,52 @@
+{
+    "name": "@hopper-ui/svgs",
+    "author": "Workleap",
+    "version": "0.0.0",
+    "description": "The svgs package.",
+    "license": "Apache-2.0",
+    "repository": {
```

Comment:
this package.json will most likely need 
```
  "exports": {
    "./*": "./dist/*"
  }
```
or something similar so we can import svgs with the path "@hopper-ui/svgs/24px/add.svg"